types/wlr_surface.c

```diff
@@ -55,6 +82,7 @@ static void surface_commit(struct wl_client *client,
 			} else {
 				uint32_t format = wl_shm_buffer_get_format(buffer);
 				wlr_texture_upload_shm(surface->texture, format, buffer);
+				wl_resource_queue_event(surface->pending_buffer, WL_BUFFER_RELEASE);
```

Comment:
i would have expected this to use wl_buffer_send_release, have never seen a manual call to wl_resource_queue_event before actually... any reasons to use this here?